sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/resources/Sgv2RowsResource.java

```diff
@@ -139,6 +142,40 @@
     return javax.ws.rs.core.Response.status(Response.Status.OK).entity(response).build();
   }
 
+  private String buildSelectAllQuery(String keyspaceName, String tableName, String fields) {
+    List<Column> columns = fieldsToColumns(fields);
+    BuiltQuery<?> qb =
+        new QueryBuilder(new SchemaNoPersistence(), null, null)
```

Comment:
So this constructor is so that we don't reference any existing schema right?

Reply:
Correct: new api/service instances do not have access to true Schema.